server/server.go

```diff
@@ -258,6 +264,16 @@ func (s *Server) absURL(pathItems ...string) string {
 	return u.String()
 }
 
+func (s *Server) connector(id string) (Connector, error) {
+	// TODO(ericchiang): Make this dynamically call the storage, possibly with
```

Comment:
@rithujohn191 hopefully this would give us a clean place to do a call to a storage.